Author: gregor herrmann <gregoa@debian.org>
Description: another error in the POD
Bug: #477399

Author: gregor herrmann <gregoa@debian.org>
Description: and some more POD errors

Index: libipc-run-perl/lib/IPC/Run.pm
===================================================================
--- libipc-run-perl.orig/lib/IPC/Run.pm	2009-07-19 03:24:36.000000000 +0200
+++ libipc-run-perl/lib/IPC/Run.pm	2009-07-19 03:26:18.000000000 +0200
@@ -112,7 +112,7 @@
 
 =head1 DESCRIPTION
 
-IPC::Run allows you run and interact with child processes using files, pipes,
+IPC::Run allows you to run and interact with child processes using files, pipes,
 and pseudo-ttys.  Both system()-style and scripted usages are supported and
 may be mixed.  Likewise, functional and OO API styles are both supported and
 may be mixed.
@@ -1005,6 +1005,8 @@
 
 =head1 ROUTINES
 
+=over
+
 =cut
 
 use strict;
@@ -3605,6 +3607,8 @@
 
 =back
 
+=back
+
 =head1 FILTERS
 
 These filters are used to modify input our output between a child
Index: libipc-run-perl/lib/IPC/Run/IO.pm
===================================================================
--- libipc-run-perl.orig/lib/IPC/Run/IO.pm	2009-07-19 03:24:36.000000000 +0200
+++ libipc-run-perl/lib/IPC/Run/IO.pm	2009-07-19 03:24:38.000000000 +0200
@@ -224,6 +224,8 @@
 
 =over
 
+=over
+
 =item filename
 
 Gets/sets the filename.  Returns the value after the name change, if
Index: libipc-run-perl/lib/IPC/Run/Timer.pm
===================================================================
--- libipc-run-perl.orig/lib/IPC/Run/Timer.pm	2009-07-19 03:24:36.000000000 +0200
+++ libipc-run-perl/lib/IPC/Run/Timer.pm	2009-07-19 03:25:44.000000000 +0200
@@ -658,6 +658,8 @@
 Note:  Setting the state to 'undef' to expire a timer will not throw an
 exception.
 
+=back
+
 =cut
 
 sub state {
@@ -675,8 +677,6 @@
 
 =pod
 
-=back
-
 =head1 TODO
 
 use Time::HiRes; if it's present.
